ani-cli

```diff
@@ -189,7 +189,7 @@ get_episode_url() {
     done
     wait
     # select the link with matching quality
-    links=$(cat "$cache_dir"/* | sed 's|^Mp4-||g' | sort -g -r -s)
+    links=$(cat "$cache_dir"/* | sed 's|^Mp4-||g;/http/!d' | sort -g -r -s)
```

Comment:
why the heck wouldn't we want encrypted traffic?

Reply:
`!` is the not operator mister

Reply:
Oh, then PR title is off?
Forcing https is okay for me

Reply:
wym, it forces the video link to contain http

Reply:
> why the heck wouldn't we want encrypted traffic?

hence my original concern

Reply:
Or does it just sharpen the link matching to include either http or https, weeding out some strange false positive?

Reply:
it matches both http and https, basically any line that contains http in it

Reply:
Fair, then it doesn't matter